cs3/storage/provider/v1beta1/provider_api.proto

```diff
@@ -724,3 +729,42 @@ message CreateHomeResponse {
   // Opaque information.
   cs3.types.v1beta1.Opaque opaque = 2;
 }
+
+message ProvisionStorageRequest {
+  // OPTIONAL.
+  // Opaque information.
+  cs3.types.v1beta1.Opaque opaque = 1;
+  // REQUIRED.
+  // The owner of the storage.
+  cs3.identity.user.v1beta1.UserId owner = 2;
```

Comment:
Let's use a full user object here, like in the Admin API, like that the Storage can have more information (like uid/gid) already provided to provision it without having to call the user service.

This function is called only when provisioning users,  the rate of this call will be very low so we can afford to prefetch this info beforehand.